### PR TITLE
Nested Markers Part 1 of a few

### PR DESCRIPTION
**Pull request by nmccready**
- more helpers
- broke up the add markers code make it more readable
- karma confs broken to be reusable to have a easy chrome launcher

### CODE REVIEW

**Comment by nmccready** [comment]
@zacronos

File: grunt/concat.js [hunk 1]
```diff
@@ -9,29 +9,9 @@ concatDist = {
     },
     src: [
         'src/directives/leaflet.js',
```
Comment (nmccready) [inline grunt/concat.js]: put what is needed to be in order first then glob it

File: test/karma-common.conf.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+var _ = require('lodash');
```
Comment (nmccready) [inline test/karma-common.conf.js]: simple config creator to keep whats common and the args are possibly what will be different

File: test/unit/markersDirectiveSpec.js [hunk 3]
```diff
@@ -80,6 +80,32 @@ describe('Directive: leaflet', function() {
         });
     });
 
+    describe('isNested',function(){
```
Comment (nmccready) [inline test/unit/markersDirectiveSpec.js]: basic spec for testing nested at the high lvl

**Comment by zacronos** [comment]
LGTM

**Pull request merged**